src/drivers/px4io/px4io.cpp

```diff
@@ -2992,19 +3004,22 @@ start(int argc, char *argv[])
 	}
 
 	bool rc_handling_disabled = false;
+	bool hitl_mode = false;
 
-	/* disable RC handling on request */
-	if (argc > 1) {
-		if (!strcmp(argv[1], "norc")) {
-
+	/* disable RC handling and/or actuator_output publication on request */
+	for (int extra_args = 1; extra_args < argc; extra_args++) {
+		if (!strcmp(argv[extra_args], "norc")) {
 			rc_handling_disabled = true;
 
+		} else if (!strcmp(argv[1], "hil")) {
```

Comment:
This should be: `strcmp(argv[extra_args], "hil"`

Reply:
Oops, thanks, I will fix. Good eye!